ee/codegen_integration/fixtures/simple_code_execution_node/code/nodes/code_execution_node/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 
 class CodeExecutionNode(BaseCodeExecutionNode[BaseState, str]):
-    filepath = "./script.py"
+    filepath = "code_execution_node/script.py"
```

Comment:
hm, as a user, I would expect the former to work. Are we sure that this is a codegen problem and not a serialization problem?

Reply:
It was a serialization problem but I wanted an easy way to know which directory the script lived in. If a user had multiple code execution nodes, I needed to know which node module it lived under since there was multiple scripts. I could be overcomplicating this thinking tho

Reply:
I suppose I find this line concerning: https://github.com/vellum-ai/vellum-python-sdks/pull/398/files#diff-c5094b531ceb9467a34a39269ca6a891fcf07152776c7f4a7c622849f2dd81ddL67

the script file should almost always be codegened relative to the current node file. push/serialization could make the same assumption. This tells me that we have something busted in the translation from local files to UI data where we lost the script we're trying to push to the UI.

Happy to talk through this more tn/tm morning!